mlrun/runtimes/databricks_job/databricks_wrapper.py

```diff
@@ -206,3 +206,6 @@ def print_status(run: Run):
 
     logger.info(f"job finished: {run.run_page_url}")
     logger.info(f"logs:\n{run_output.logs}")
+    run_output_dict = run_output.as_dict()
```

Comment:
in case of exception in the outer try-catch, I think run_output is not defined

Reply:
In the outer try-catch there is only try and finally. If there will be an error, it will not catch it, and will raise it (python will also run the finally part in this scenario).

so this code will not run in the case of exception....

Reply:
add comment.